src/main/java/SYSC4806/Controller/HomeController.java

```diff
@@ -31,4 +39,19 @@ public String home(Model model) {
         model.addAttribute("programs", programRepository.findAll());
         return "index";
     }
+
+    @GetMapping("/results/{program}/{year}")
```

Comment:
Just thinking does this make more sense to just be a natural query string parameter as opposed to a path variable? I don't see much merit into saying something like "http://localhost:8080/results/1/3". It's not very intuitive from what you're doing here, versus something that is like "http://localhost:8080/results?program=1&year=3"

Reply:
Since only a portion of the page reloads, it actually stays at "/". This mapping is just used to get the program and year from the view in a convenient way.

Reply:
Okay so from what I understand is it's mostly just for convenience because the Select values already have the numeric/string representation of program and year. Well I don't know a better way to do it so it seems okay to me -shrug-